src/components/Person/Person.tsx

```diff
@@ -26,10 +26,16 @@ const PersonPresentation: React.FunctionComponent<Props> = ({ item }) => {
 
 
   //// Affiliation needs work in the API
```

Comment:
See my other comment on Person Metadata, but perhaps do this already in the container code